java/vector/src/main/java/org/apache/arrow/vector/TimeMilliVector.java

```diff
@@ -18,7 +18,7 @@
 
 package org.apache.arrow.vector;
 
-import io.netty.buffer.ArrowBuf;
+import siren.io.netty.buffer.ArrowBuf;
```

Comment:
Does that mean we weren't always using the custom changes of netty we did ?

Reply:
Yes, for vector, but this was fixed after this release 0.8.0